packaging/language/bundler.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Tim Hoiberg <tim.hoiberg@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+DOCUMENTATION='''
+---
+module: bundler
+short_description: Manage Ruby Gem dependencies with Bundler
+description:
+  - Manage installation and Gem version dependencies for Ruby using the Bundler gem
+version_added: "2.0.0"
+options:
+  executable:
+    description:
+      - The path to the bundler executable
+    required: false
+  state:
+    description:
+      - The desired state of the Gem bundle. C(latest) updates gems to the most recent, acceptable version
+    required: false
+    choices: [present, latest]
+    default: present
+  chdir:
+    description:
+      - The directory to execute the bundler commands from. This directoy needs to contain a valid Gemfile or .bundle/ directory
+    required: false
+  exclude_groups:
+    description:
+      - A list of Gemfile groups to exclude during operations. This only applies when state is C(present). Bundler considers this a 'remembered' 
+      property for the Gemfile and will automatically exclude groups in future operations even if C(exclude_groups) is not set
+    required: False
+  extra_args:
```

Comment:
Keep this, but we prefer to have the actual options as parameters in the module that translate into the args to the tool